.github/workflows/dockerfile_pull_request_verifier.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Build Dockerfile
+on:
+  pull_request:
+    paths:
+      - ./Dockerfile
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run local registry
+        run: docker run -d -p 5000:5000 registry:2
+
+      - name: Build and push image
+        uses: docker/build-action@v1
+        env:
+          DOCKER_BUILDKIT: 1
+        with:
+          registry: localhost:5000
+          repository: zerobaseio/smart-tracing
+          tags: pull_request
+
+      - name: Remove local image
+        run: docker image rm localhost:5000/zerobaseio/smart-tracing:pull_request
+
+      - name: Run image from registry
+        run: docker run localhost:5000/zerobaseio/smart-tracing:pull_request
```

Comment:
I don't think these actions are needed since the build agent resets after every build.

Reply:
Hi @jasonspriggs, the idea of these two steps is just to check if the image was successfully pushed and can be pulled from the registry.
This doesn't fail too much, so I'm fine removing it if you prefer.